### DISCUSSION TITLE
How to bundle all classes in tailwind without minification

### DISCUSSION THREAD

**Discussion by Ezetalent** [tailwindlabs/tailwindcss#9437] [Help]
Initially, when i bundle tailwind and not in production mode, it bundles all CSS available and export, but I just tried it today and it could not generate any CSS unless I have the utility class available in my HTML files.

**Comment by wongjn** [comment]
Are you using Tailwind v2 or v3? In Tailwind v3, only classes you use in files covered by the `content` globs in the Tailwind config should get generated, whether in development or production.

**Reply by Ezetalent** [reply]
I'm using Tailwind v3. 
Thanks for the clarification
